### ISSUE TITLE
reponse null

### ISSUE THREAD

**Issue by SergioMordente** [weldonsouza/cielo_ecommerce#10]
Estou testando conforme o exemplo e o retorno sempre é null. Este pacote esta sendo usado? Alguém passou por este problema?

**Comment by weldonsouza** [comment]
até agora não, sabe informar se suas chaves "merchantId" e "merchantKey" estão ok?

**Comment by SergioMordente** [comment]
Sim, as chaves estão corretas. Uso os memos dados em uma aplicação em Java e funciona corretamente. Estou testando em produção.
